backend/core/detection.py

```diff
@@ -33,6 +33,7 @@
     ["S", "-", "-"],
     ["E", "0", "-"],
     ["E", "1", "-"],
+    ["E", "N", "-"],
```

Comment:
do not forget to update the regexes also on the frontend so that we can search for this new type.

https://github.com/FORTH-ICS-INSPIRE/artemis/blob/master/frontend/webapp/views/main/templates/hijacks.htm#L368

Reply:
Also update the Type-N documentation here https://github.com/FORTH-ICS-INSPIRE/artemis/blob/master/frontend/webapp/static/js/custom/display_info.js#L37

For now state that we detect type-Ns due to no-export route leaks from the neighbor of the first-hop neighbor upstream ("fake" link non-compliant to policy)

Reply:
gonna remove the letter and leave it as `N` as it is covered